src/asdf_standard/integration.py

```diff
@@ -0,0 +1,29 @@
+from pathlib import Path
+import sys
+
+if sys.version_info < (3, 9):
+    import importlib_resources
+else:
+    import importlib.resources as importlib_resources
+
+from asdf.resource import DirectoryResourceMapping
+
+import asdf_standard
+
+
+def get_resource_mappings():
+    resources_root = importlib_resources.files(asdf_standard) / "resources"
+    if not resources_root.is_dir():
+        # In an editable install, the resources directory will exist off the
+        # repository root:
+        resources_root = Path(__file__).absolute().parent.parent.parent / "resources"
+        if not resources_root.is_dir():
+            raise RuntimeError("Missing resources directory")
+
+    return [
+        DirectoryResourceMapping(resources_root / "schemas" / "stsci.edu", "http://stsci.edu/schemas/", recursive=True),
+        DirectoryResourceMapping(
+            resources_root / "manifests" / "asdf-format.org" / "core",
+            "asdf://asdf-format.org/core/extensions/",
```

Comment:
The manifest URI prefix is actually `asdf://asdf-format.org/core/manifests/` but this is still passing because we have a resource plugin in asdf itself that is pulling from the submodule.

Any thoughts on this directory structure vs `asdf-format.org/manifests`?  That feels more natural to me since the authority is the root level name, and it will save us from having to add another DirectoryResourceMapping when we eventually have asdf-format.org schemas.  In any case we should pick a firm convention for that and make all the schemas repos match since like you pointed out we're not consistent right now.

Reply:
I like the `asdf-format.org/manifests`.

Reply:
See 48800a6 for the change to `core/manifests/`.